data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1875 @@ mission "Stone of our Fathers 5"
+		ship "Bactrian" "DTU Endeavor"
+		ship "Bulk Freighter" "DTU Consistency"
+		ship "Bulk Freighter" "DTU Dependable"
+		ship "Freighter" "DTU Empathy"
+	on visit
+		dialog
+			`You have reached <planet>, but the Transport Union ships have not all arrived yet. You should take off and wait for them to catch up with you.`
+			
+	on complete
+		payment 850000
+		event "skadenga death" 367
+		conversation
+			`For the occasion of the primitives' arrival Helsingor has remade itself into a mad circus. There are impromptu bands playing, random unplanned parades, teeming crowds of excited neighbors holding up signs welcoming the Skadenga into the 31st century. There is also every kind of emergency and civil personnel available here. Firemen, police, doctors, social workers; all claiming, testing, prodding and questioning the dazed immigrants.`
+			`	The Ondurdis finds you in the chaos and lets you know she wishes to see the God-Stone. Together you travel to the mall, and you explain to her how the structure was built around it, using it as an accent wall to the various posh establishments within.`
+			`	Upon seeing this the old and overwhelmed matron falls to her knees in tears.`
+			scene "scene/buildings"
+			`	"These folk are lost." She says. "We are lost."`
+			`	She looks at you with her tear-wrecked face and says,"You are instrument of Skade. I saw it. But why? Why this? Why to us?"`
+			`	After that she will say no more to you, she lifts her eyes to the sky and begins to chant wildly. A few Skadenga who have made it this far join her. Various news outlets swarm around them to record the Ondurdis' breakdown.`
+			`	You back away slowly, and bump into someone. You turn to see the real-estate agent Carla Von Hausen smiling at you.`
+			`	"Well done, Captain <last>. I've been authorized to tell you that my superiors have deposited 850,000 credits into your account, as was agreed upon earlier."`
+			choice
+				`	"Your superiors??"`
+			`	Carla grins. "The Census and Culture Bureau. I am one of their field agents. They are most pleased with how you handled this. The Skadenga have been a problem for centuries, and you solved it."`
+			`	She pats you on the back. "Anyways, I just wanted to pass that on. I have lots of work to do, thanks to you." She winks and wanders into the crowd.`
+			`	You return to your ship to count your money.`
+			
+			
+			
+event "skadenga death"
+
+
+
+mission "Skadenga Call 1"
+	landing
+	name "Check on Hjlod"
+	description `Fly to <destination>, and see what Hjlod and the Skadenga need from you.`
+	source
+		government "Republic"
+	destination "Asgard"
+	to offer
+		has "Home for Skadenga 7: done"
+		has "event: skadenga death" 
+	on offer
+		dialog
+			`	There is a new message in your inbox.`
+			`	<first>, we need your help. Come to Asgard.`
+			`			Signed, Hjlod`
+	on complete
+		dialog
+			`Upon landing you receive another message in your inbox that simply says 'spaceport'.`
+
+
+
+mission "Skadenga Call 2"
```

Comment:
This whole mission rubs me up wrong, The Deep is the standout example of possibly the only human society in ES that actually seems to care about the rights of people and the heritage of its past.

While I think it's pretty reasonable that the different people involved in trying to get them onto Asgard are leveraging a profit incentive to save the budget or just because it looks good for whatever politician is in charge, I don't think it's reasonable to go down the path of having then dismantled the community and abandoned the uncooperatives. If there's anyone left in the ES universe that appreciates the need to maintain communities to effectively assimilate them, it would be the Deep.

Perhaps this should be a hard-core idealist subset of them that just can't abide the "here to remind the people of their roots" narrative, and feel certain that their true place is elsewhere?

They could still be very disappointed in you, and maybe they're willing to burn down some modernist thing that particularly hurts their community? (Maybe the mall?) So you still have a source of conflict to report them on.

After all, a modern effort to "help" can still be misguided and dangerously ineffective without being actively malicious.

But perhaps just being able to take those hard-core idealists either back to Nifel, or on to Windblain might be viable, rather than rescuing them from a humanitarian nightmare.

Reply:
First, I am going to quote MZ here: 
_"It’s worth keeping in mind that pretty much every species or faction in Endless Sky lies to you, either because they have their own agenda or because they don’t know what the heck they’re talking about (or both). Sifting out what’s actually true is part of the fun, and my intention was to have different people tell you obviously contradictory stories about the same things._"

I would argue that my depiction of the Deep as flawed here falls directly in line with his world-building philosophy. If anything it brings them closer to MZ's ideals. To certain extent this also a small matter, you are dealing with a junior member of the Census and Culture Bureau, not its leadership, and in a case like this it is easy for a population of a few thousand to just fall between the cracks. The society, and maybe top leadership as a whole can certainly still be 'good guys'. But that doesn't mean that everyone throughout the entire bureaucracy is a saint. In the places where no one is looking, you're still going to have people being people. 

This is also is art imitating life, if you see how 'modern' societies deal with immigrants, or aboriginal populations (Native Americans/First Peoples in North America, Aboriginals in Australia, Sami in Scandinavia) etc. Even when these societies mean well, they often do significant damage, and someone in power nearly always has ulterior motives. 

The one thing though, is that we're dealing with 50 people out of 2200 who immigrated, I suppose we could show that many of those people did transition to their new culture fine. But, I am cautious of doing that for thematic reasons. The 'story' of this line is all about consequences. The player often makes decisions that effect people, and then flies away after the mission end without having face those decisions again. And this all about them meeting those consequences.

Reply:
Yeah I'm not proposing that the rest of them all transitioned "fine", but just weren't badly enough off that the idealists would choose living in a dank alleyway wearing rags and evading law enforcement as a preferable alternative... like that's waaaaaaaayyyy darker than it needs to be. 

As an Australian, I'm quite familiar with the damage from meaning well but not understanding, that still happens today but in a more modern context. This present writing sounds like the 1970's "let's separate the children from their families and raise them civilised" approach, whereas the level I'm advocating for is the more modern version.
The modern version obviously still has many issues (public housing that separates them from their cultural practices so they choose to not live in it, education programs that teach in the dominant language to "help them learn" but leads to failure, etc.) but it's more 'callous neglect' or 'public relations posturing' and less 'actual humanitarian nightmare'.

Reply:
Alright, you've convinced me. I'll start thinking about a more nuanced and less grimdark approach to this =)

Reply:
I have re-written some of the mission. Hjlod now points out that most of the Skadenga are happy. The government's actions on Asgard are clearly from a help-first approach. I also re-wrote some of Carla's dialog to have her more sympathetic and less condescending. I think it now appears less malicious and more accidental/well-intentioned.

Reply:
You actually went slightly nicer than I expected, but yes I think this is a much more appropriate tone. Threw in one suggestion in my new review but you can mark this thread as resolved.